src/components/Tokens/TokenDetails/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { SwapAction } from './types'
+
+/**
+ * Displays the time as a human-readable string.
+ *
+ * @param {number} timestamp - Transaction timestamp in milliseconds.
+ * @param {number} locale - BCP 47 language tag (e.g. en-US).
+ * @returns {string} Message to display.
+ */
+export function getLocaleTimeString(timestamp: number, locale = 'en-US') {
+  const now = Date.now()
```

Comment:
probably would have been easier to test (no need to manipulate timers) with this api

```
export function getLocalTimeString(timestamp: number, locale = 'en-US', getNow = Date.now) {
  const now = getNow()
```